bin/seed-deck.js

```diff
@@ -39,7 +39,5 @@ const files = fs.readdirSync(program.path);
       description: details.Description 
     }, { new: true, upsert: true, returnNewDocument: true });
   }
-
-  console.log(`Upserted ${files.length} cards in "${program.name}"`);
-  process.exit();
+  console.log(`Upserted ${image_files.length} cards in "${program.name}"`);
 })();
```

Comment:
nit: newline at end of file to pass `.eslint` rules (if we don't have that rule, we should add it)